cmd/state/internal/cmdtree/ppm.go

```diff
@@ -171,25 +177,79 @@ func addOtherCommands(cmds []*captain.Command) []*captain.Command {
 			"config",
 			"configuration settings",
 			func(_ *captain.Command, _ []string) error {
-				return printDefault()
+				return printDefault(prime)
 			},
 		),
 		captain.NewShimCommand(
 			"gui",
 			"opens the graphical user-interface",
 			func(_ *captain.Command, _ []string) error {
-				return printDefault()
+				return printDefault(prime)
 			},
 		),
 	)
 }
 
-func addInfoCommand(cmds []*captain.Command) []*captain.Command {
+func addInfoCommand(prime *primer.Values, cmds []*captain.Command) []*captain.Command {
 	return append(cmds, captain.NewShimCommand(
 		"info",
 		"prints ppm help message",
 		func(_ *captain.Command, _ []string) error {
-			return printMain()
+			return printMain(prime)
 		},
 	))
 }
+
+func shim(prime *primer.Values, intercepted, replaced, localeID string, args ...string) error {
+	err := shim(prime, intercepted, replaced, localeID, args...)
```

Comment:
`shim` is calling `shim`? Did you test this?

Reply:
Whoops that was a mistake. Fixed now in the runner